be/src/exec/file_scanner.cpp

```diff
@@ -355,4 +358,109 @@ Status FileScanner::create_random_access_file(const TBrokerRangeDesc& range_desc
     }
 }
 
+void merge_schema(const std::vector<std::vector<SlotDescriptor>>& input, std::vector<SlotDescriptor>* output) {
+    if (output == nullptr) {
+        return;
+    }
+
+    std::vector<std::shared_ptr<SlotDescriptor>> merged_schema;
+    std::map<std::string, size_t> merged_schema_index;
+    for (const auto& schema : input) {
+        for (const auto& slot : schema) {
+            auto itr = merged_schema_index.find(slot.col_name());
```

Comment:
deal with the case of column name? sensitive or insensitive？

Reply:
It's case-senstitive now.